vaxrank/cli.py

```diff
@@ -73,10 +74,41 @@ def cached_run_arg_parser():
         default="",
         help="Path to JSON file containing results of vaccine peptide report")
     add_output_args(arg_parser)
+    add_optional_output_args(arg_parser)
     add_supplemental_report_args(arg_parser)
     return arg_parser
 
 
+# Lets the user specify whether they want to see particular sections in the report.
+def add_optional_output_args(arg_parser):
+    manufacturability_args = arg_parser.add_mutually_exclusive_group(required=False)
+    manufacturability_args.add_argument(
+        "--include-manufacturability-in-report",
+        dest="manufacturability",
+        action="store_true")
+    manufacturability_args.add_argument(
+        "--no-manufacturability-in-report",
+        dest="manufacturability",
+        action="store_false")
+    arg_parser.set_defaults(manufacturability=True)
```

Comment:
Remind me, how does this interact with `required=True` for the group? Will the user still have to supply one of the two args, even though there's a default?